javascript/javascript1/week1/Ahousepriceestimator.js

```diff
@@ -0,0 +1,37 @@
+// Project 3
+//A house price estimator
+let houseprice;
+let volumeinMeters;
+
+// 1-peter's house price calculation
+let gardenSizeInM2 = 100;
+let width = 8;
+let depth = 10;
+let height = 10;
+let actualhouseprice = 2500000;
+
+volumeinMeters = width * depth * height;
+houseprice = volumeinMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+
+if (actualhouseprice > houseprice) {
+  console.log("NO Peter, you are paying too much");
+} else {
+  console.log("Go for it Peter , it is reasonable;)");
+}
```

Comment:
What will be logged if the price paid is just right? Do you think you need an extra `console.log` for it?